sky/cli.py

```diff
@@ -4367,23 +4379,38 @@ def serve_down(service_names: List[str], all: bool, purge: bool, yes: bool):
         raise click.UsageError(
             'Can only specify one of SERVICE_NAMES or --all. '
             f'Provided {argument_str!r}.')
+    replica_id_is_defined = replica_id is not None
+    if replica_id_is_defined and len(service_names) != 1:
+        raise click.UsageError(
+            'Must specify one and only one service when replica ID is '
+            'specified.')
```

Comment:
```suggestion
        raise click.UsageError(
            'Must specify one and only one service '
            'when replica ID is specified.')
```
nit: try to squash them into 2 lines (pls run formatter).